src/components/auth-required/notice/Modal.tsx

```
@@ -0,0 +1,36 @@
+
+      {data?.posts?.map(({ content, created_at, id, title }) => (
+        <div key={id}>
+          <div className="flex items-center gap-3">
+            <h3 className="text-TEXT-MAIN text-T4">{title}</h3>
+            <h4 className="text-TEXT-ALT text-T5">
+              {created_at.split('T')[0]}
+            </h4>
+          </div>
+          <div
+            className="text-TEXT-MAIN text-ST5"
+            dangerouslySetInnerHTML={{ __html: content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)